reana_server/reana_admin/check_workflows.py

```diff
@@ -165,54 +181,135 @@ def check_workflows(
             fg="red",
         )
         logging.exception(error)
-        sys.exit(1)
+        return [], [], total_workflows
```

Comment:
Returning empty arrays causes the command to print `OK` and exit with code zero. What do you think about throwing exceptions (or not catching them)?